readiness-tool/vendors.json

```diff
@@ -0,0 +1,627 @@
+{
+  "apps": {
+    "AD EBiS": {
+      "category": [
```

Comment:
Are there any vendors that are both "Ads" and "Analytics" categories? If not, this doesn't need to be an array.